### PR TITLE
Add graph, include to release-notes

### PR DESCRIPTION
**Pull request by gayaldassanayake**
## Purpose
$Subject

### CODE REVIEW

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md [hunk 1]
```diff
@@ -103,6 +103,22 @@ To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://g
 
 #### New features
 
+##### CLI
+
+Added new CLI command `bal graph` which resolves the dependencies of the current package and prints the dependency graph in the console. This produces the textual representation of the dependency graph using the DOT graph description language.
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:108]:
```suggestion
Added a new `bal graph` CLI command, which resolves the dependencies of the current package and prints the dependency graph in the console. This produces the textual representation of the dependency graph using the DOT graph description language.
```

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md [hunk 2]
```diff
@@ -103,6 +103,22 @@ To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://g
...
+
+```
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:110]:
```suggestion
```ballerina
```

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md [hunk 3]
```diff
@@ -123,4 +139,20 @@ To view bug fixes, see the GitHub milestone for 2201.2.0 (Swan Lake) of the repo
 - [update tool](https://github.com/ballerina-platform/ballerina-update-tool/issues?q=is%3Aissue+milestone%3A%22Ballerina+2201.2.0%22+is%3Aclosed+label%3AType%2FBug)
 - [OpenAPI](https://github.com/ballerina-platform/openapi-tools/issues?q=is%3Aissue+label%3AType%2FBug+milestone%3A%22Ballerina+2201.2.0%22+is%3Aclosed)
 
+### Ballerina packages updates
+
+#### New features
+
+Added `include` field under the `[package]` table. It allows providing paths to any additional resources, which need to be packed in the BALA file.
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:146]:
```suggestion
Added an `include` field under the `[package]` table. It allows providing paths to any additional resources, which need to be packed in the BALA file.
```

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md [hunk 4]
```diff
@@ -123,4 +139,20 @@ To view bug fixes, see the GitHub milestone for 2201.2.0 (Swan Lake) of the repo
...
+
+```
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:148]:
```suggestion
```ballerina
```

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md [hunk 5]
```diff
@@ -103,6 +103,22 @@ To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://g
 
 #### New features
 
+##### CLI
+
+Added a new `bal graph` CLI command, which resolves the dependencies of the current package and prints the dependency graph in the console. This produces the textual representation of the dependency graph using the DOT graph description language.
```
Comment (azinneera) [inline release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:108]:
```suggestion
Introduced `bal graph` CLI command, which resolves the dependencies of the current package and prints the dependency graph in the console. This produces the textual representation of the dependency graph using the DOT graph description language.
```

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md [hunk 6]
```diff
@@ -123,4 +139,20 @@ To view bug fixes, see the GitHub milestone for 2201.2.0 (Swan Lake) of the repo
 - [update tool](https://github.com/ballerina-platform/ballerina-update-tool/issues?q=is%3Aissue+milestone%3A%22Ballerina+2201.2.0%22+is%3Aclosed+label%3AType%2FBug)
 - [OpenAPI](https://github.com/ballerina-platform/openapi-tools/issues?q=is%3Aissue+label%3AType%2FBug+milestone%3A%22Ballerina+2201.2.0%22+is%3Aclosed)
 
+### Ballerina packages updates
+
+#### New features
+
+Added an `include` field under the `[package]` table. It allows providing paths to any additional resources, which need to be packed in the BALA file.
```
Comment (azinneera) [inline release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:146]:
```suggestion
Introduced `include` field under the `[package]` table in `Ballerina.toml`. It allows providing paths to any additional resources, which need to be packed in the BALA file.
```

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md [hunk 7]
```diff
@@ -123,4 +139,20 @@ To view bug fixes, see the GitHub milestone for 2201.2.0 (Swan Lake) of the repo
...
+
+```ballerina
+[package]
+org = "samjs"
+name = "winery"
+version = "0.1.0"
+include = [documents/”, "images/sample.png"]
+```
+
+**Info:** The `include` field accepts a string array, which contains the directory or file paths to include in the BALA. The paths should be relative to the package root directory.
```
Comment (azinneera) [inline release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:156]: Shall we have this part in the description as suggested in the comment https://github.com/ballerina-platform/ballerina-release/pull/2027/files#r913165929? @praneesha WDYT?

Comment (gayaldassanayake) [reply release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md]: Removed **Info** and added that content according to the comment https://github.com/ballerina-platform/ballerina-release/pull/2027/files#r913165929.

**Review by azinneera — APPROVED** [review]

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md [hunk 8]
```diff
@@ -103,6 +103,22 @@ To view bug fixes, see the [GitHub milestone for 2201.2.0 (Swan Lake)](https://g
 
 #### New features
 
+##### CLI
+
+Introduced `bal graph` CLI command, which resolves the dependencies of the current package and prints the dependency graph in the console. This produces the textual representation of the dependency graph using the DOT graph description language.
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:108]:
```suggestion
Introduced the `bal graph` CLI command, which resolves the dependencies of the current package and prints the dependency graph in the console. This produces the textual representation of the dependency graph using the DOT graph description language.
```

File: release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md [hunk 9]
```diff
@@ -123,4 +139,18 @@ To view bug fixes, see the GitHub milestone for 2201.2.0 (Swan Lake) of the repo
 - [update tool](https://github.com/ballerina-platform/ballerina-update-tool/issues?q=is%3Aissue+milestone%3A%22Ballerina+2201.2.0%22+is%3Aclosed+label%3AType%2FBug)
 - [OpenAPI](https://github.com/ballerina-platform/openapi-tools/issues?q=is%3Aissue+label%3AType%2FBug+milestone%3A%22Ballerina+2201.2.0%22+is%3Aclosed)
 
+### Ballerina packages updates
+
+#### New features
+
+Introduced `include` field under the `[package]` table in `Ballerina.toml`. It accepts a string array of paths to any additional files and directories, which need to be packed in the BALA file. The path should be relative to the package root directory.
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md:146]:
```suggestion
Introduced an `include` field under the `[package]` table in `Ballerina.toml`. It accepts a string array of paths to any additional files and directories, which need to be packed in the BALA file. The path should be relative to the package root directory.
```
Was it intentional to have `package` written within square brackets?

**Review by praneesha — APPROVED** [review]

**Pull request merged**